### ISSUE TITLE
Path to learn orgmode in neovim for users without experience

### ISSUE THREAD

**Issue by AdrianRibao** [nvim-orgmode/orgmode#320]
### Does this feature exist in Emacs orgmode core?

N/A

### Feature value

I'm a vim user since many years ago, I moved to neovim a long time ago and I recently updated my `vimrc` with Lua configs.

I've no experience with emacs. Nothing.

I have read and watched videos about org mode. I'm really interested, so the first thing I did was look for some plugin in vim for emulating org mode. I have found this project, and it's exciting, but I don't know how to start.

I'm reading org mode manual, and I also had a look at [the doc](https://github.com/nvim-orgmode/orgmode/blob/master/doc/orgmode.txt), but some things I'd like to have in this project are a small tutorial, or a resource reference for learning more about org-mode, specifically using this plugin.

It would be nice to have a FAQ in the [main plugin page](https://nvim-orgmode.github.io/index.html). For example:
* What is org mode
* Where can I learn more about org mode
* How can I use org mode as a productivity tool
* How to use the agenda, todos or any other topic

I don't know where to start.

**Comment by jgollenz** [comment]
Adding more info about orgmode and related links / resources is a good idea. But I am against putting it (and a FAQ section) into the README.md, except as a link (like we do for the docs). It is already quite long.

Concerning the tutorial: what do have in mind here? An example workflow where you create a `todo.org` and we explain how to add headlines, TODO states, agenda & capture, archiving, ... ?

**Comment by AdrianRibao** [comment]
Yes, I agree with not putting the FAQs in the README, but in the [page](https://nvim-orgmode.github.io/index.html), that's what I meant.
It would be nice to have a nice "Getting started" in the [vim docs](https://github.com/nvim-orgmode/orgmode/blob/master/doc/orgmode.txt)

For the tutorial, that's exactly what I meant. I don't know even how to start, how to organize the files, or how to find the right mapping. There are a lot of mappings, I'm totally lost. It would be nice to have the most used mapping in the "getting started" section.

I already saw an issue about improving mappings explanation #314. In my case, I can open and write an `org` file but I don't know how to be proficient and use the keys and commands that this plugin provides.

I'm speaking as someone that knows nothing about orgmode. I guess there are other people like me out there :-)

I know you can organize `org` files as you like. There are many "frameworks" for the organization, like "Getting Things Done" and many others. But I would find an explanation of a basic layout helpful, and maybe add at the end of the section references to articles/books/videos explaining other methodologies.

Thanks!

**Comment by jgollenz** [comment]
Sorry, I misread that. Yes, I think it would fit well on the homepage. 

I'll require some time to think of how to layout such a tutorial. Do you have a list of things that should absolutely be in there in your opinion? I take many things for granted with the plugin now, you know best where a beginner needs help. What are the things you are trying to do?

**Comment by AdrianRibao** [comment]
Sure, I will prepare an outline. I'm learning many things already, and I think I could write the most problematic barriers new users find when getting started with org mode.

Do you want me to create a PR in `nvim-orgmode.github.io` so I can write the outline, the things I have learned and let the community and the experts fill the gaps I miss?

BTW, the more I learn about this plugin, the most amazing I find it!

**Comment by jgollenz** [comment]
Yeah, a PR in `nvim-orgmode.github.io` sounds good. Looking forward to it :+1:

**Comment by gerazov** [comment]
There's a Getting started section here https://nvim-orgmode.github.io/getting_started.html that could be the place for this.

Related to #183.
